CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+## Pull Request Guidelines
+
+External contributions to Alt-Ergo may be proposed using git's standard pull request mechanism.
+The following terms apply to all such contributions:
+
+* We require all pull requests to pass :
+
+	* Our non regression suite, you can run `make non-regression` to check if your changes affect the performance and answers of Alt-Ergo.
+
+	* Our style checking, every lines should length 80 columns maximum and should be indent with ocp-indent.
+	  We strongly recommand to use the following command line so the indent and style checking will be done automatically when commiting.
+
+	```cp rsc/extra/pre-commit-git-hook .git/hooks/pre-commit```
+
+* Pull request should be rebased and up to date from the `base` branch to be merged.
```

Comment:
This is quite ambiguous, people could try and look for a branch named `base` to make PRs against. One way to avoid the ambiguity is to introduce the concept of the `base`branch in the first paragraph when you specify that contributions are welcome as PRs, agains some base branch (which would typically be `next`, some particular dev branch, or in rare cases `master`).

Reply:
OK, i will wait for your suggestion and commit it then, thanks.